test/cmd/newapp.sh

```diff
@@ -126,6 +158,10 @@ os::cmd::expect_success 'oc delete template ruby-helloworld-sample'
 os::cmd::expect_success_and_text 'oc new-app mysql --name=db' 'db'
 os::cmd::expect_success 'oc new-app https://github.com/openshift/ruby-hello-world -l app=ruby'
 os::cmd::expect_success 'oc delete all -l app=ruby'
+# check for error when template JSON file has errors
+jsonfile="${TMPDIR:-/tmp}/foo.json"
+echo '{"test"}' > "${jsonfile}"
```

Comment:
I searched for `TMPDIR` in the project, but didn't find any place that would set it to anything other than `/tmp`.
I'm assuming this file will not be deleted by the test framework after run, is it to be considered a problem?